cyvcf2/cyvcf2.pyx

```diff
@@ -11,7 +11,10 @@ import numpy as np
 from array import array
 import math
 import ctypes
-
+try:
+  from pathlib import Path
+except ImportError:
+  from pathlib2 import Path  # python 2 backport
```

Comment:
I suppose for python2 this makes `pathlib` or `pathlib2` a dependency?  Can we just not support python2 after a certain version of cyvcf2?  It would make things so much nicer, especially if we can use the typing library for python 3?

Reply:
we need to keep python2 support.